doc/ipython-notebooks/clustering/GMM.ipynb

```diff
@@ -318,17 +307,14 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "def estimate_gmm(X, num_components):\n",
     "    # bring data into shogun representation (note that Shogun data is in column vector form, so transpose)\n",
-    "    features=RealFeatures(X.T)\n",
```

Comment:
for the sake of minimal changes, i would not delete this and do inline below, but rather replace the line with the factory